commit ca737d0824ee9ccbcc836bbcb6779188876602b0 (refs/changes/97/1497/1)
Author: Joshua M. Clulow <jmc@joyent.com>
Date:   2017-02-10T23:20:10+00:00 (2 years, 8 months ago)
    
    MANTA-3161 sdc-manta.git should run catest with the right node

diff --git a/Makefile b/Makefile
index 74b9d2f..316845a 100644
--- a/Makefile
+++ b/Makefile
@@ -92,7 +92,7 @@ check:: $(NODE_EXEC)
 
 .PHONY: test
 test: | $(CATEST)
-	$(CATEST) -a
+	PATH="$(TOP)/build/node/bin:$$PATH" $(CATEST) -a
 
 $(CATEST): deps/catest/.git
 
